lib/forms/submission_statuses/error_handler.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Forms
+  module SubmissionStatuses
+    class ErrorHandler
+      SOURCE = 'Lighthouse - Benefits Intake API'
+      TITLE_PREFIX = 'Form Submission Status'
+
+      def handle_error(response)
+        errors = parse_error(response.status, response.body)
```

Comment:
Refactored by passing `status` and `body` instead of the entire `response` object.  This should reduce this method's dependency on the `response` object.